src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -20,10 +20,29 @@ const ParameterField = (props: ParameterFieldProps) => {
   const handleChange = useCallback(
     (value: any) => {
       if (parameter.multiple && (value === null || value === undefined)) {
-        onChange([]);
+        return onChange([]);
       }
-      if (parameter.type === "int" && value !== "") {
-        onChange(parseInt(value, 10));
+
+      if (parameter.type === "int") {
```

Comment:
I think it's better to always take the value of the input instead of trying to deal with the int value and the string value.